saos-persistence/src/main/java/pl/edu/icm/saos/persistence/common/InitializingVisitor.java

```diff
@@ -19,15 +19,19 @@ public void visit(DataObject dataObject) {
     
     public void visit(CommonCourtJudgment judgment) {
         initializeJudgment(judgment);
-        JpaUtils.initialize(judgment.getCourtDivision());
-        JpaUtils.initialize(judgment.getCourtDivision().getCourt());
+        if (judgment.getCourtDivision() != null) {
```

Comment:
NPE in case of trying to fetch judgment from repository that was created by TestInMemoryCcObjectFactory.createSimpleCcJudgment();